Updates to charRNN examples

shiffman opened the pull request:
### → Step 1:  Which branch are you submitting to? 🌲
Now to main!!

### → Step 2: Describe your Pull Request 📝
These are some small updates to the charRNN examples

* I limited the temperature slide to a minimum of 0.01, a 0 temperature is invalid and ml5 processes that as 0.5.
* For the interactive example, the sketch was failing when the input box was empty and it would often not generate based on the most up to date text. I am not sure of a great way to solve this but I have a function that runs every 500 milliseconds and generates new text if there has been an update but the user is not actively still typing.

```javascript
function setup() {
  // Check every so often if we should generate something new
  setInterval(checkGenerate, 500);
}

// Has 500 milliseconds passed since the last time a change was made?
function checkGenerate() {
  let passed = millis() - last;
  if (passed > 500 && !generated) {
    generate();
    generated = true;
  }
}
```

bomanimc commented on examples/p5js/CharRNN/CharRNN_Interactive/sketch.js:
```diff
@@ -28,19 +31,45 @@ function setup() {
   tempSlider = select('#tempSlider');
 
   // Run generate anytime something changes
```
Looks like we might want to update this comment

bomanimc reviewed (approved):
Looks solid to me! How do you feel about these lint errors?

shiffman commented:
I updated the incorrect comment. In terms of the lint errors I tend to employ a "p5 friendly" style which often goes against JS conventions. But as `const` is used in other places throughout the examples I adjusted for the linting to. pass. Maybe this is a larger conversation at some point about the "style" of our examples. Here are some questions:

* Do we use `const` and `let` or only `let`?
* Do we use anonymous functions passed directly as callbacks or separately defined named functions?
* Do we move towards using `async` and `await` instead of callbacks?

bomanimc commented:
Yeah I think these would be great discussions to have as well. I would love to understand more about how these topics are being taught in 2020. I'm also happy to change our linter to allow us to move towards more beginner-friendly approaches. On the other side of things, I sort of feel like it would probably make most sense for us to only break modern JS conventions in cases where we feel like it creates significant confusion. For example, maybe const vs. let is pretty easy to understand, but arrow functions are confusing enough to warrant a break in pattern. 

Happy to chat about this further in the future!
